source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -115,48 +123,95 @@ Table 3.p (below): List of Data_Type Values
      - Reports
 
    * - Article
-     - An article, typically published in a journal or reference work.
+     - An article from a journal, or an article available as a standalone content item not aggregated into a title, for example in an institutional repository.
```

Comment:
For Article the Parent_Data_Type always should be Journal, so this can no longer be correct.

Reply:
There are plenty of situations where an article is a standalone item - preprints in repositories for example - and would not have a parent data type. Definition is correct.

Reply:
What I'm trying to say is that a publication with a Data_Type that has a Parent_Data_Type somehow must be related to that Parent_Data_Type, otherwise the Data_Type should not be used.